image/image.go

```diff
@@ -48,9 +48,11 @@ type V1Image struct {
 // Image stores the image configuration
 type Image struct {
 	V1Image
-	Parent  ID        `json:"parent,omitempty"`
-	RootFS  *RootFS   `json:"rootfs,omitempty"`
-	History []History `json:"history,omitempty"`
+	Parent     ID        `json:"parent,omitempty"`
+	RootFS     *RootFS   `json:"rootfs,omitempty"`
+	History    []History `json:"history,omitempty"`
+	OSVersion  string    `json:"os_version,omitempty"`  // os_version similar to docker_version; BIKESHED: osversion, os.version
+	OSFeatures []string  `json:"os_features,omitempty"` // BIKESHED: os_features, os.features
```

Comment:
Since we went with `os.version` and `os.features` in https://github.com/docker/distribution/pull/1534, can we use the same naming convention here?